spec/features/media_object_spec.rb

```diff
@@ -21,6 +21,17 @@
     @user = FactoryGirl.create(:administrator)
     login_as @user, scope: :user
   end
+  before :all do
```

Comment:
Why does the intercom config need to be here?  I'm nervous that we will never test without them and miss errors or bake in assumptions.